src/main/java/org/jabref/gui/ai/components/privacynotice/PrivacyNoticeComponent.fxml

```diff
@@ -44,6 +44,15 @@
                   </Text>
                </children>
             </TextFlow>
+             <TextFlow fx:id="geminiPrivacyTextFlow">
+                 <children>
+                     <Text text="%If you have chosen the Google Gemini as AI provider, the privacy policy of Google Gemini applies. You find it at %0.">
```

Comment:
Can't we unify the l10n here and add the name of the AI provider as a parameter as well?
If you have chosen %0 as AI provider, the privacy policy of Google Gemini applies. You find it at %1.">

Reply:
![image](https://github.com/user-attachments/assets/ab5fd96a-ac94-476b-9783-1b5b6728bb59)

OMG! All of this time, when user clicks on privacy policy of an AI provider, they're sent to API URL!

Ahah, I'll fix this